targets/mediatek-filogic

```diff
@@ -0,0 +1,20 @@
+-- ASUS
```

Comment:
We don't add board-support in base-version updates.

Reply:
jup, needs to be dropped obviously (also AFAIK nobody tested those devices following the checklist), but since we're realistically months away from this switch happening i would keep it for now as it means that i don't need to track diffrent branches. (I'll put a "DROP FOR MERGE" into the Commit Message next time i'll force push.)

Reply:
This can be weeks if not days away.

Reply:
Seems like your more optimistic in regards to when v2023.1.x will be branched off. I've dropped the commits.